### PR TITLE
gui.extension.Element doc fixed

### LINKED ISSUE #1819: [DOCS] Updating docs on Element definition
### Issue Description

Correcting the properties definition in the Element

### Screenshots or Examples (if applicable)

_No response_

### Proposed Solution (optional)

Change `properties (List[ElementProperty]): The list of properties for this element. ` to `properties (Dict[ElementProperty]): The dictionary of properties for this element.`

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by namnguyen20999**
resolves https://github.com/Avaiga/taipy/issues/1819

### CODE REVIEW

File: taipy/gui/extension/library.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def __init__(
 
         Arguments:
             default_property (str): the name of the default property for this element.
-            properties (List[ElementProperty]): The list of properties for this element.
+            properties (Dict[ElementProperty]): The dictionary of properties for this element.
```
Comment (FabienLelaquais) [inline taipy/gui/extension/library.py]: Hum.
You will have to say a word about what the keys and values must be...

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
Tell me more!

File: taipy/gui/extension/library.py [hunk 2]
```diff
@@ -112,8 +112,8 @@ def __init__(
         *react_component* is ignored.
 
         Arguments:
-            default_property (str): the name of the default property for this element.
-            properties (List[ElementProperty]): The list of properties for this element.
+            default_property (str): The name of the default property for this element.
+            properties (Dict[ElementProperty]): The dictionary containing the properties of this element, where the keys are the property names and the values are instances of ElementProperty.
```
Comment (FredLL-Avaiga) [inline taipy/gui/extension/library.py]: that would be Dict[str, ElementProperty] ?

**Review by FabienLelaquais — DISMISSED** [review]
With Fred's remark, that's fine!

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**